knowledge-base/using-filters-in-fiddler-everywhere-for-advanced-search.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Using Filters in Fiddler Everywhere for Advanced Search
+description: Learn how to use filters in Fiddler Everywhere to search for specific values within request and response bodies and headers.
+type: how-to
+page_title: Using Filters in Fiddler Everywhere for Advanced Search
+slug: using-filters-in-fiddler-everywhere-for-advanced-search
+tags: fiddler-everywhere, search, filters, request body, response body, headers
+res_type: kb
+---
+
+## Environment
+
+|----------|-------|
+| Product  | Fiddler Everywhere |
+| Version  | 5.0.0 and above |
+
+## Description
+
+Fiddler Everywhere's global search option is designed to search only within the HTTP headers of captured traffic. This is because the application is optimized for performance and user experience. Searching through the potentially large  HTTP body content in each captured session would be inefficient and slow.
```

Comment:
```suggestion
Fiddler Everywhere's global search option is designed to search only within the HTTP headers of captured traffic. This is because the application is optimized for performance and user experience. Searching through the potentially large HTTP body content in each captured session would be inefficient and slow.
```